vllm/v1/engine/core.py

```diff
@@ -344,6 +379,18 @@ def process_input_socket(self, input_path: str):
                 request_type = type_frame.buffer
                 request_data = data_frame.buffer
 
+                if request_type == EngineCoreRequestType.STATS.value:
```

Comment:
This is where the backend process responds. Note this is independent from the actual execution thread.